cfme/automate/explorer/method.py

```diff
@@ -306,35 +312,44 @@ class MethodEditView(AutomateExplorerView):
     embedded_method = EntryPoint(locator='//*[@id="automate-inline-method-select"]//button',
                                  tree_id="treeview-entrypoint_selection")
 
+    ansible_max_ttl = Input(name='provisioning_execution_ttl')
+
     save_button = Button('Save')
     reset_button = Button('Reset')
     cancel_button = Button('Cancel')
 
     def before_fill(self, values):
         location = self.context['object'].location.lower()
-        if 'display_name' in values and location in ['inline', 'playbook']:
-            values['{}_display_name'.format(location)] = values['display_name']
+        if 'display_name' in values:
```

Comment:
@ganeshhubale What about the `playbook` case? Won't this fail if `location=='playbook'`?

Reply:
Hey @john-dupuy 
Actually I am not getting why this question. So I tested **test_automate_ansible_playbook_method_type_crud** < this test case with these changes locally. And it works as expected.

Reply:
@ganeshhubale I have some test that uses this method that has failed in this section of code before. I remember I had to add in the `location in ['inline', 'playbook']` line to get it to work. Can you please try running `test_alert_run_ansible_playbook`?

My point is that you've removed the code that handles the case where `location == 'playbook'`. I.e. to keep the original code intact, L324 should be
```python
if location in ['inline', 'playbook']:
```

Does that make sense? Or am I missing something here?